includes/migration/javascript-v8/browser-other-changes.mdx

```diff
@@ -0,0 +1,19 @@
+### Removal of `@sentry/replay` package
+
+The `@sentry/replay` package is no longer required. Instead you can import the relevant methods directly from your SDK. In addition to this, the integration is now functional instead of class based.
```

Comment:
```suggestion
The `@sentry/replay` package is no longer required. Instead you can import the relevant methods directly from your SDK. In addition to this, the integration is now functional instead of class-based.
```